wordpress-mobile/WordPress-Android
# [Blogging prompts v1] Implement "My Site" card analytics events

Fixes #16521 

To test:
- Make sure blogging prompts feature flag is enabled
- With a site selected, tap on "HOME" tab
- Scroll to prompts card
  - `"blogging_prompts_my_site_card_answer_prompt_tapped"` event should be triggered when "Answer prompt" button is tapped
  - `"blogging_prompts_my_site_card_share_tapped"` event should be triggered when "Share" button is tapped
  - `"blogging_prompts_my_site_card_menu_tapped"` event should be triggered when card overflow menu is tapped
  - `"blogging_prompts_my_site_card_menu_view_more_prompts_tapped"` event should be triggered when menu item "View more prompts" is tapped
  - `"blogging_prompts_my_site_card_menu_skip_this_prompt_tapped"` event should be triggered when menu item "Skip this prompt" is tapped
  - `"blogging_prompts_my_site_card_menu_remove_from_dashboard_tapped"` event should be triggered when menu item "Remove from dashboard" is tapped

The actions to undo "Skip this prompt" and undo "Remove from dashboard" were not implemented yet, so the analytics events are not part of this PR.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and unit testing

3. What automated tests I added (or what prevented me from doing so)
Added `BloggingPromptsCardAnalyticsTrackerTest.kt` and updated `MySiteViewModelTest.kt`

PR submission checklist:
